README.md

```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
 - Topoplot series
```

Comment:
Circular topoplots are missing

also the text above is suboptimal, we should remove the lines

The toolbox requires a Makie-Backend to be loaded, e.g. `using CairoMakie`
Some plotting functions require data-frames, with fields like `:estimate` or `:time`.

Or at least send them down to the quick start or something